### PR TITLE
batched email

### PR DESCRIPTION
**Pull request by otakup0pe**
With this PR email is no longer sent at guest creation, it instead will be sent "out of band" via a django management command. The batch size is configurable and retries are implicit as the email needs to successfully send before it's marked as being sent. This PR also introduces the first (of a handful) of custom management commands.

- [x] additional testing of `send_onboarding` command
- [x] install shell script scaffolding / cron job on remote host

### CODE REVIEW

File: backend/reservations/management/commands/send_onboarding.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+logging.basicConfig(stream=sys.stdout, level=roombaht_config.LOGLEVEL)
+logger = logging.getLogger('SendOnboarding')
+
+def onboarding_email(email, otp):
+    hostname = my_url()
+    body_text = f"""
+        BleepBloopBleep, this is the Room Service RoomBaht for Room Swaps letting you know the floors have been cleaned and you have been assigned a room. No bucket or mop needed.
+
+        After you login below you can view your current room, look at other rooms and send trade requests. This functionality is only available until Monday 11/7 at 5pm PST, so please make sure you are good with what you have or trade early.
+
```
Comment (mpesavento) [inline backend/reservations/management/commands/send_onboarding.py:17]: @otakup0pe @Index01 
Let's put some content in here about how people are expected to navigate the room swap:
```
1. Find your desired room with the same room type (eg queen).
2. Click "SendSwapRequest" and enter in information about how you'd like that room to contact you, eg email address or phone #.
3. They will contact you. Or not.
4. If you both agree to swap rooms, login (link below) and find the room you want to swap under "My Rooms".
5. Click CreateSwapCode. Give this swap code to the other person.
6. Within 10 minutes, they need to go to their page, click "EnterSwapCode" for the room you want, and enter the swap code.
7. Magic intradimensional elves will hop around and inform you of your success. And you will have the room you always wanted!
Really, if you have a friend that you want to swap rooms with, just get their swap code and enter it for your released room.
```

File: backend/reservations/management/commands/send_onboarding.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+logging.basicConfig(stream=sys.stdout, level=roombaht_config.LOGLEVEL)
+logger = logging.getLogger('SendOnboarding')
+
+def onboarding_email(email, otp):
+    hostname = my_url()
+    body_text = f"""
+        BleepBloopBleep, this is the Room Service RoomBaht for Room Swaps letting you know the floors have been cleaned and you have been assigned a room. No bucket or mop needed.
+
+        After you login below you can view your current room, look at other rooms and send trade requests. This functionality is only available until Monday 11/7 at 5pm PST, so please make sure you are good with what you have or trade early.
```
Comment (mpesavento) [inline backend/reservations/management/commands/send_onboarding.py:16]: Let's change the date for this years date:
`only available until Sunday 11/5 at 5pm PST`
Still checking on confirmation for this date/time.

File: backend/reservations/management/commands/send_onboarding.py (hunk 1 above)
Comment (otakup0pe) [reply backend/reservations/management/commands/send_onboarding.py:17]: Just copied the email text we already had. Not a bad idea to clean it up for this year / clarity. If you get a moment, def push text changes here.

File: backend/reservations/management/commands/send_onboarding.py (hunk 2 above)
Comment (mpesavento) [reply backend/reservations/management/commands/send_onboarding.py:16]: confirmed and changed

File: backend/reservations/management/commands/send_onboarding.py (hunk 1 above)
Comment (mpesavento) [reply backend/reservations/management/commands/send_onboarding.py:28]: pushed updated copy

**Comment by Index01** [comment]
lewks goud! hashtag send it.

**Comment by otakup0pe** [comment]
well that was a ride.....

**Pull request merged**